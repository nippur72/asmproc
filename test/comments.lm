TEXT TEXT TEXT TEXT TEXT TEXT 
TEXT TEXT TEXT TEXT  /* COMMENTED */ TEXT TEXT TEXT TEXT 

TEXT TEXT TEXT TEXT TEXT TEXT 
TEXT TEXT TEXT TEXT  /* 
COMMENT 
COMMENT 
COMMENT 
COMMENT 
*/ TEXT TEXT TEXT TEXT 

TEXT // COMMENT 

TEXT ; COMMENT
