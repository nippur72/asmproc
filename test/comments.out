TEXT TEXT TEXT TEXT TEXT TEXT 
TEXT TEXT TEXT TEXT                  TEXT TEXT TEXT TEXT 

TEXT TEXT TEXT TEXT TEXT TEXT 
TEXT TEXT TEXT TEXT     
        
        
        
        
   TEXT TEXT TEXT TEXT 

TEXT // COMMENT 

TEXT ; COMMENT
